tests/pkg_tweetnacl/Makefile

```
@@ -0,0 +1,15 @@
+include ../Makefile.tests_common
+
+BOARD_INSUFFICIENT_MEMORY := arduino-duemilanove \
+                             arduino-uno \
+                             nucleo-f031k6 \
+                             #
+
+TEST_ON_CI_WHITELIST += native
+
+CFLAGS += -DTHREAD_STACKSIZE_MAIN=\(4*THREAD_STACKSIZE_DEFAULT\)
```

Using the stack size value used in `unittests` works without crash:

```suggestion
CFLAGS += -DTHREAD_STACKSIZE_MAIN=\(4*THREAD_STACKSIZE_DEFAULT+THREAD_EXTRA_STACKSIZE_PRINTF\)
```

---

I've incremented the "4" to "5".

---

Agreed if you put the comment in the commit too as mentioned in https://github.com/RIOT-OS/RIOT/pull/10185#discussion_r226865550